docs/deploying-to-azure.md

```diff
@@ -40,13 +40,18 @@ LOCATION="eastus2"
 # Container Apps
 LOG_ANALYTICS_WORKSPACE="super-heroes-logs"
 CONTAINERAPPS_ENVIRONMENT="super-heroes-env"
-# Kafka
-EVENTHUB_NAMESPACE="super-heroes-eventhub"
-EVENTHUB_TOPIC="fights"
 # Postgres
 POSTGRES_DB="super-heroes-db"
 POSTGRES_DB_ADMIN="superheroesadmin"
 POSTGRES_DB_PWD="super-heroes-p#ssw0rd-12046"
+# MongoDB
+MONGO_DB="fights-db"
+# Kafka
+KAFKA_NAMESPACE="fights-kafka"
```

Comment:
Should add something in here (like the user's username) so that its unique.

Maybe `KAFKA_NAMESPACE="fights-kafka-$(whoami)"`?